tsconfig.spec.json

```diff
@@ -1,17 +1,18 @@
-{
-  "extends": "./tsconfig.base.json",
-  "compilerOptions": {
-    "outDir": "./out-tsc/spec",
-    "types": [
-      "jasmine"
-    ]
-  },
-  "files": [
-    "src/test.ts",
-    "src/polyfills.ts"
-  ],
-  "include": [
-    "src/**/*.spec.ts",
-    "src/**/*.d.ts"
-  ]
-}
+/* To learn more about this file see: https://angular.io/config/tsconfig. */

```

Comment:
Again, please double check.
Could just be something screwy with Github's syntax highlighter.